src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridView.PropertyGridViewAccessibleObjectTests.cs

```diff
@@ -32,6 +32,20 @@ public void PropertyGridViewAccessibleObject_Ctor_NullOwner_ThrowsArgumentNullEx
             Assert.Throws<ArgumentNullException>(() => new PropertyGridViewAccessibleObject(null, null));
         }
 
+        [WinFormsTheory]
+        [InlineData(/*NavigateDirection.Parent*/ 0)]
+        [InlineData(/*NavigateDirection.NextSibling*/ 1)]
+        [InlineData(/*NavigateDirection.PreviousSibling*/ 2)]
+        [InlineData(/*NavigateDirection.FirstChild*/ 3)]
+        [InlineData(/*NavigateDirection.LastChild*/ 4)]
```

Comment:
I would made like this:
```suggestion
        [InlineData((int)NavigateDirection.Parent)]
        [InlineData((int)NavigateDirection.NextSibling)]
        [InlineData((int)NavigateDirection.PreviousSibling)]
        [InlineData((int)NavigateDirection.FirstChild)]
        [InlineData((int)NavigateDirection.LastChild)]
```